### PR TITLE
Theme buttons

### PR DESCRIPTION
**Pull request by AndersonCeci**
made hover on dark mode buttons also added seperate code for theme buttons

### CODE REVIEW

File: src/app/settings/settings.component.html [hunk 1]
```diff
@@ -36,110 +36,13 @@ <h5>Pamja e Zgjedhur</h5>
                                 }
                             </div>
                             <div
-                                class="theme-selector"
+                                class="theme-selector theme-selector-settings"
                                 [ngClass]="{
-                                    'theme-selector-dark-mode':
+                                    'theme-selector-dark-mode theme-selector-settings-dark-mode':
```
Comment (OpenCovenant) [inline src/app/settings/settings.component.html:41]: `theme-selector-settings` and `theme-selector-settings-dark-mode` seem to be declared in the `app-dark-mode-buttons` component instead, why is that?

File: src/app/settings/settings.component.html [hunk 2]
```diff
@@ -36,110 +36,13 @@ <h5>Pamja e Zgjedhur</h5>
                                 }
                             </div>
                             <div
-                                class="theme-selector"
+                                class="theme-selector theme-selector-settings"
                                 [ngClass]="{
-                                    'theme-selector-dark-mode':
```
Comment (AndiBraimllari) [inline src/app/settings/settings.component.html:41]: `theme-selector-settings` and `theme-selector-settings-dark-mode` seem to be declared in the `app-dark-mode-buttons` component instead, why is that?

File: src/app/settings/dark-mode-buttons/dark-mode-buttons.component.css [hunk 3]
```diff
@@ -0,0 +1,178 @@
+.option-set-system-mode-light-mode.active {
```
Comment (AndiBraimllari) [inline src/app/settings/dark-mode-buttons/dark-mode-buttons.component.css:1]: Ideally, everything declared here is only used by this component

**Comment by AndiBraimllari** [comment]
Please run prettier

File: src/app/settings/settings.component.html (hunk 2 above)
Comment (AndersonCeci) [reply src/app/settings/settings.component.html:41]: The classes theme-selector-settings and theme-selector-settings-dark-mode are specifically used for mobile view styling in the settings section. Here's why:

1. In the settings section:

- On mobile devices, the theme selector buttons are arranged in a column layout.
- We use theme-selector-settings for this mobile-specific layout.
- theme-selector-settings-dark-mode is used for the dark mode variant of this mobile layout.


2. In the header:

- The theme selector maintains its original layout, even on mobile devices.
- It doesn't need a special class for mobile view, as its appearance doesn't change.

By adding "settings" to the class names (theme-selector-settings), we create separate styling rules for the mobile view of the settings section, without affecting the header's theme selector.

**Pull request merged**